packages/@aws-cdk/custom-resource-handlers/lib/core/nodejs-entrypoint-handler/index.ts

```diff
@@ -95,7 +96,7 @@ function renderResponse(
   };
 }
 
-async function submitResponse(status: 'SUCCESS' | 'FAILED', event: Response) {
+export async function submitResponse(status: 'SUCCESS' | 'FAILED', event: Response) {
```

Comment:
Make `submitResponse` public so we can reuse it in the CrossAccountRecordSet custom resource handler.